api/pom.xml

```diff
@@ -9,5 +9,13 @@
 	<packaging>jar</packaging>
 	<name>rwandahivflowsheet API</name>
 	<description>API project for rwandahivflowsheet</description>
-
```

Comment:
We should aim to keep all omod/web dependencies in the omod project, and have no such dependencies in the api project.  So this should be moved to the omod pom.xml

Reply:
The issue is the `org.openmrs.module.web.extension.LinkExt` class which is used by `RwandaAdultAndPediHIVFlowsheetFormItem`, `RwandaAdultHIVFlowsheetFormItem` and  `RwandaPediHIVFlowsheetFormItem` classes from the API project. I get compilation errors in those classes when I remove the legacy-ui dependency from the API project.

Reply:
@rubailly - yes, well all of those FormItem classes that extend LinkExt should also be in the omod project.  Just move these into a package in the omod project, and change config.xml (which is also in omod) to refer to these new packages, and things should work fine.

Reply:
[Done](https://github.com/PIH/openmrs-module-rwandahivflowsheet/pull/2/commits/2b97a7e318a90cefe799f949a6644364654aec0a)